diff --git a/src/mochijson2_fork.erl b/src/mochijson2_fork.erl
index 355f068..0da135f 100644
--- a/src/mochijson2_fork.erl
+++ b/src/mochijson2_fork.erl
@@ -91,6 +91,7 @@
 -define(IS_WHITESPACE(C),
         (C =:= $\s orelse C =:= $\t orelse C =:= $\r orelse C =:= $\n)).
 
+-type(json_term() :: any()).
 -type(decoder_option() :: any()).
 -type(handler_option() :: any()).
 
@@ -100,8 +101,6 @@
 -type(json_object() :: {struct, [{json_string(), json_term()}]}).
 -type(json_eep18_object() :: {[{json_string(), json_term()}]}).
 -type(json_iolist() :: {json, iolist()}).
--type(json_term() :: json_string() | json_number() | json_array() |
-                    json_object() | json_eep18_object() | json_iolist()).
 
 -record(encoder, {handler=null,
                   utf8=false}).
